### PR TITLE
sslh: Enable libcap support

### PR DESCRIPTION
**Pull request by th0ma7**
_Motivation:_  Request from user to enable `libcap` support in `sslh`
_Linked issues:_  #3714

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by th0ma7** [comment]
Pending user testing to confirm it meets requirement at #3714

File: cross/sslh/Makefile [hunk 1]
```diff
@@ -8,19 +8,24 @@ PKG_DIST_SITE = https://github.com/yrutschle/sslh/archive
...
 
-DEPENDS = cross/libconfig cross/pcre
+DEPENDS = cross/libconfig cross/pcre cross/libcap
 
 HOMEPAGE = http://www.rutschle.net/tech/sslh.shtml
 COMMENT  = sslh accepts HTTPS, SSH and OpenVPN connections on the same port
 LICENSE  = GPLv2
 
+PRE_CONFIGURE_TARGET = sslh_pre_configure
 CONFIGURE_TARGET = nop
```
Comment (ymartin59) [inline cross/sslh/Makefile:18]: As configure target is skipped, I propose to switch `USELIBCAP` variable as `sslh_configure` target

**Review by ymartin59 — APPROVED** [review]

File: cross/sslh/Makefile (hunk 1 above)
Comment (th0ma7) [reply cross/sslh/Makefile]: Good call, done.

**Comment by th0ma7** [comment]
@ymartin59 while at it I've also looked at updating `libcap` and while at it add 2x of the static binaries namely `getpcaps` and `getcap`.

**Comment by th0ma7** [comment]
But I'm seriously wondering what is benefit of this?  It really ain't much of a trouble to add but what value does it bring?
```
# ps -ef | grep sslh
nobody   25697     1  0 16:34 ?        00:00:00 /volume1/@appstore/sslh/bin/sslh -F/volume1/@appstore/sslh/var/sslh.cfg
nobody   25699 25697  0 16:34 ?        00:00:00 /volume1/@appstore/sslh/bin/sslh -F/volume1/@appstore/sslh/var/sslh.cfg
# ./getpcaps 25697
25697: =
# ./getpcaps 25699
25699: =
# ./getcap ../bin/sslh 
# ./getcap -v ../bin/sslh 
../bin/sslh
```
```

**Review by ymartin59 — APPROVED** [review]

**Pull request merged**